tests/test_input_manager.py

```diff
@@ -102,3 +103,110 @@ def test_load_data_raises_exception(mock_input_manager: InputManager) -> None:
     with patch("builtins.open", mock_open_func):
         with pytest.raises(Exception):
             mock_input_manager._load_data("bad/path.csv")
+
+
+@pytest.fixture
+def mock_metadata(mocker) -> Dict[str, Dict[str, Any]]:
```

Comment:
```suggestion
def mock_metadata(mocker: MockerFixture) -> Dict[str, Dict[str, Any]]:
```
Add type annotations for the mocker object here and for the others unit tests.